lib/appium_lib/core/android/touch.rb

```diff
@@ -0,0 +1,85 @@
+module Appium
+  module Core
+    module Android
+      module Touch
+        def self.extend_touch_actions
+          ::Appium::Core::TouchAction.class_eval do
+            # Visible for testing
+            # @private
+            def swipe_coordinates(end_x: nil, end_y: nil, offset_x: nil, offset_y: nil)
```

Comment:
use `offset` and measure `end`(absolute value) in this method to make more helpful for users like Java client.